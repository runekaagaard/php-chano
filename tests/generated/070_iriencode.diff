001- S%C3%B8r-Tr%C3%B8ndelag
001+ Fatal error: Call to undefined method DtlIter::iriencode() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/070_iriencode.php on line 4