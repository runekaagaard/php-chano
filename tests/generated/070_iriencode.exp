S%C3%B8r-Tr%C3%B8ndelag