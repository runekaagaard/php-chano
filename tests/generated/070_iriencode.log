
---- EXPECTED OUTPUT
S%C3%B8r-Tr%C3%B8ndelag
---- ACTUAL OUTPUT
Fatal error: Call to undefined method DtlIter::iriencode() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/070_iriencode.php on line 4
---- FAILED
