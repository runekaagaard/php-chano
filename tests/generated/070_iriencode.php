<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'Sør-Trøndelag')));
foreach ($items as $i) echo $i->input->iriencode();
