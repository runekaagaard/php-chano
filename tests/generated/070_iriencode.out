Fatal error: Call to undefined method DtlIter::iriencode() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/070_iriencode.php on line 4